proxy/http/HttpTransact.cc

```diff
@@ -63,7 +63,12 @@ static char range_type[] = "multipart/byteranges; boundary=RANGE_SEPARATOR";
 
 extern HttpBodyFactory *body_factory;
 
-static const char local_host_ip_str[] = "127.0.0.1";
+inline static bool
+is_localhost(const char *name, int len)
```

Comment:
I should say, I'm fine punting on that for later, just file a separate Jira on unifying "localhost" comparisons.